Modules/Quotas/src/Quotas.Application/Tiers/Commands/SeedQueuedForDeletionTier/Handler.cs

```diff
@@ -27,7 +27,8 @@ public async Task Handle(SeedQueuedForDeletionTierCommand request, CancellationT
         }
 
         var metrics = await _metricsRepository.FindAll(CancellationToken.None);
-        queuedForDeletionTier.AddQuotaForAllMetricsOnQueuedForDeletion(metrics);
+        queuedForDeletionTier.AddQuotaForAllMetricsOnQueuedForDeletion(metrics.Where(m => m.Key != MetricKey.NumberOfCreatedDatawalletModifications
+                                                                                          && m.Key != MetricKey.NumberOfStartedDeletionProcesses));
```

Comment:
You should move the `&&` into the first line. We have it like this in other places as well.

Reply:
Btw: I just meant the `&&` and not the the whole line. ;) So that `m.key` and `m.Key´ are right below each other. This makes it smooth to read (at least in my opinion 😃)